frameworks/kafka/universe/config.json

```diff
@@ -71,6 +61,26 @@
             "type": "string",
             "enum":  ["OFF", "FATAL", "ERROR", "WARN", "INFO", "DEBUG", "TRACE", "ALL"],
             "default": "INFO"
+          },
+          "security": {
+            "type": "object",
+            "properties": {
+                "tls": {
```

Comment:
Why no nesting per security type here? (I really dislike these long json sections ... )

Reply:
How does the UI handle a third layer of nesting?

Reply:
I spoke to Dima from foundation about this, and it basically flattens everything (I also tested with some pretty deep nesting, and this seems to be the case).